### ISSUE TITLE
Change `fees` type of `ContractMessageRequest`

### ISSUE THREAD

**Issue by luis-stakeordie** [stakeordie/griptape.js#80]
Change the type of `fees` property to be `number` and calculate the fees based on the following function:

```js
const gasPriceUscrt = 0.25;
export function getFeeForExecute(gas: number): StdFee {
  return {
    amount: [
      { amount: String(Math.floor(gas * gasPriceUscrt) + 1), denom: "uscrt" },
    ],
    gas: String(gas),
  };
}
```

So then, you can do:

```js
const stkd = {
  messages: {
    stake() {
      const handleMsg = { stake: { } };
      const fees = 650000;
      return { handleMsg, fees };
    }
  }
}
```

**Comment by jsandovalg** [comment]
To solve this issue my approach is:

1. Into "ContractMessageRequest " Interface change type from StdFee  to number.
2. Create a  function into utils.ts file called "getFeeForExecute" and export it.
3. Use the new getFeeForExecute in  createContract function into index.ts file from contracts .

**Comment by luis-stakeordie** [comment]
> To solve this issue my approach is:
> 
> 1. Into "ContractMessageRequest " Interface change type from StdFee  to number.
> 2. Create a  function into utils.ts file called "getFeeForExecute" and export it.
> 3. Use the new getFeeForExecute in  createContract function into index.ts file from contracts .

Awesome, go ahead.

**Comment by luis-stakeordie** [comment]
Closed by #81
